.github/workflows/ci.yml

```diff
@@ -0,0 +1,116 @@
+name: Continuous Integration
```

Comment:
Hey @nightscape I was not actively looking at the project but thanks for the PR. Will it still work or need some changes?  I've not really used the Github actions so I'll need to figure this out.

Reply:
Hi @mayur2810, the GitHub action is still working:
https://github.com/nightscape/sope/actions/runs/11456230896